serialization.md

```diff
@@ -285,6 +237,18 @@ We need to interperet a `ByteArray` as a `String` again so that we can call `Kec
     rule #unparseDataByteArray( DATA ) => replaceFirst(Base2String(#asInteger(#asByteStack(1) ++ DATA), 16), "1", "0x")
 ```
 
+- `#addrBytes` Takes an Account and represents it as a 20-byte wide ByteArray (or an empty ByteArray for a null address)
+- `#wordBytes` Takes an Int and represents it as a 32-byte wide ByteArray
+
+```k
+    syntax ByteArray ::= #addrBytes( Account ) [function]
+                       | #wordBytes( Int )     [function]
+ // -----------------------------------------------------
+    rule #addrBytes(.Account) => .ByteArray
+    rule #addrBytes(ACCT)     => #padToWidth(20, #asByteStack(ACCT))
+    rule #wordBytes(WORD)     => #padToWidth(32, #asByteStack(WORD))
```

Comment:
Should these have validity constraints on the inputs? Like `#rangeAddress(ACCT)` and `#rangeUInt(256, WORD)`?

Reply:
Sounds like a good idea. It could help catch some bugs in the future.